joint_state_controller/test/test_load_joint_state_controller.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 PAL Robotics SL.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <memory>
+
+#include "controller_manager/controller_manager.hpp"
+
+#include "test_robot_hardware/test_robot_hardware.hpp"
+
+#include "rclcpp/utilities.hpp"
+
+TEST(TestLoadJointStateController, load_controller)
+{
+  rclcpp::init(0, nullptr);
+
+  std::shared_ptr<test_robot_hardware::TestRobotHardware> robot =
+    std::make_shared<test_robot_hardware::TestRobotHardware>();
+
+  robot->init();
+
+  std::shared_ptr<rclcpp::executor::Executor> executor =
+    std::make_shared<rclcpp::executors::SingleThreadedExecutor>();
+
+  controller_manager::ControllerManager cm(robot, executor, "test_controller_manager");
+
+  ASSERT_NO_THROW(cm.load_controller("test_joint_state_controller",
```

Comment:
```suggestion
  ASSERT_NO_THROW(
    cm.load_controller(
      "test_joint_state_controller",
      "joint_state_controller/JointStateController"));
```

uncrustify (master) is complaining about this.